db/migrate/20170609140748_create_votes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateVotes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :votes do |t|
+      t.integer :user_id
+      t.integer :votable_id
+      t.string :votable_type
+      t.timestamps
+    end
+    add_index :votes, :votable_id
+    add_index :votes, :votable_type
```

Comment:
I'd probably add a compound index here with a uniqueness constraint so that a user cannot have more than one vote for the same votable thing.  Check out the migration docs to get a sense for how to write a compound migration.  You've already done indices with uniqueness constraints (adding the `unique: true` option to the `add_index` call), but hit me up if you have questions and we can walk through it.